src/oge/helpers.py

```diff
@@ -229,7 +253,8 @@ def create_plant_ba_table(year):
             retired_bas["retirement_date"].dt.year < year, "ba_code"
         ].unique()
     )
-    # if there are any plants that have been assigned to a retired BA, set its BA code as missing
+    # if there are any plants that have been assigned to a retired BA, set its BA code
+    # as missing
     plant_ba.loc[plant_ba["ba_code"].isin(retired_bas), "ba_code"] = np.NaN
```

Comment:
To be consistent with the deleted code above, we should use `pd.NA` here

Reply:
Done